CORE-11 Add swagger docs :description-file param.

psarando opened the pull request:
This PR will add the `:description-file` docs param to `common-swagger-api.routes`, and bump the version to `2.10.0-SNAPSHOT`.

slr71 commented on src/common_swagger_api/routes.clj:
```diff
@@ -1,4 +1,10 @@
+
+(defmethod compojure.api.meta/restructure-param :description-file
+  [_ path acc]
+  (update-in acc [:swagger] assoc :description (slurp (io/resource path))))
```
LOL, I completely forgot about moving this function here. I'm glad you remembered. 😆

psarando replied:
Since this is a notable new feature to this lib, I think I'll also cut a `2.10.0` release with this method, then merge the rest of this PR in a `2.10.1-SNAPSHOT` version.

slr71 reviewed (approved):
LGTM :shipit:

psarando committed:
CORE-11 Add swagger docs `:description-file` param

Add the `:description-file` docs param to `common-swagger-api.routes`.
Bumped minor version.

psarando commented:
Thanks for the review! I'll begin bumping versions then merging these PRs now 👍
